components/colors/index.css

```
@@ -8,55 +8,79 @@
 /* Gray */
 
 :root {
-  --gray-1: #f9f9fb;
-  --gray-2: #e9e9e9;
-  --gray-3: #d9d9d9;
-  --gray-4: #c9c9c9;
-  --gray-5: #b9b9b9;
-  --gray-6: #a9a9a9;
-  --gray-7: #696969;
-  --gray-8: #2f2f2f;
-  --gray-9: #1b1b1b;
+  --color-gray-1: #f9f9fb;
+  --color-gray-2: #e9e9e9;
+  --color-gray-3: #d9d9d9;
+  --color-gray-4: #c9c9c9;
+  --color-gray-5: #b9b9b9;
+  --color-gray-6: #a9a9a9;
+  --color-gray-7: #696969;
+  --color-gray-8: #2f2f2f;
+  --color-gray-9: #1b1b1b;
```

I'd suggest to use 10/20/30/* to make it more clear that these refer to percentages (?!):

```suggestion
  --color-gray-10: #f9f9fb;
  --color-gray-20: #e9e9e9;
  --color-gray-30: #d9d9d9;
  --color-gray-40: #c9c9c9;
  --color-gray-50: #b9b9b9;
  --color-gray-60: #a9a9a9;
  --color-gray-70: #696969;
  --color-gray-80: #2f2f2f;
  --color-gray-90: #1b1b1b;
```